Automattic/wp-calypso
# Add/new email comparison card props

#### Changes proposed in this Pull Request

Create a new component "EmailProvidersComparisonStacked" placeholder to wire things up.

Modify EmailPromoCard to match Saygun's design
- Add a new property to inject a header component (in addition to the old title string)

Modify Price component to add a new property "additional information"

Create a new EmailProviderStackedCard to provide the Saygun's modifications (features inside of the card, logo at the end of it)

### Work in progress
![image](https://user-images.githubusercontent.com/5689927/144249083-f1841e8f-a2b0-455c-86ac-bc761f74a7b5.png)

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison` (it is only 

**Remarks**: visible through the path /email/{domain}/purchase/{site}?flags=emails/new-email-comparison

You should see the beginning of a new component being developed.

Related to 1200182182542585-as-1201414571816235

[Subsequent PR](https://github.com/Automattic/wp-calypso/pull/58400)
